app/src/main/java/mollie/tictactoe/GameHelper.java

```diff
@@ -0,0 +1,58 @@
+package mollie.tictactoe;
+
+import mollie.tictactoe.players.MobilePlayer;
+import ttt.game.Board;
+import ttt.game.GameEngine;
+import ttt.game.Marks;
+
+public class GameHelper {
```

Comment:
I'm not sold on this abstraction.
The suffix "-Helper" indicates that it has no obvious purpose other than supporting other classes that do the real work. 
createGame() also made me suspicious... and getGame()... Is that a singleton? Those were used _once upon a time in the west_. Maybe there is a way to restructure this?